src/src/Commands/Settings.cpp

```diff
@@ -42,11 +42,12 @@ String Command_Settings_Unit(struct EventStruct *event, const char* Line)
 
 String Command_Settings_Name(struct EventStruct *event, const char* Line)
 {
-	return Command_GetORSetString(event, F("Name:"),
-				      Line,
-				      Settings.Name,
-				      sizeof(Settings.Name),
-				      1);
+	Command_GetORSetString(event, F("Name:"),
+												 Line,
+												 Settings.Name,
+												 sizeof(Settings.Name),
+												 1);
+	return Settings.getUnitname();
```

Comment:
Is this still the same behavior?
GetOrSet.... functions typically return "Ok" or something similar when setting a value, or don't they?

Reply:
Reverted that change